### DISCUSSION TITLE
Newly introduced log message verbosity level

### DISCUSSION THREAD

**Discussion by jeger-at** [seaweedfs/seaweedfs#5407] [Q&A]
Just a quick question to understand the newly introduced verbose log messages in log_read.go. Are they intentionally set to verbosity level 0, so except you decide to totally skip logging with -v=-1 they're flooding log files quite a lot. Just saw they were defined with V(4) before adding further details, now it's quite chatty.

e.G. https://github.com/seaweedfs/seaweedfs/blob/master/weed/util/log_buffer/log_read.go#L60

**Comment by chrislusf** [comment]
fixed. thanks for reporting.

**Reply by jeger-at** [reply]
@chrislusf thanks for the fast reply.. I'm quite confident there are more than just this example.. 
There is for example another one I quickly found which also appears in our logs:
https://github.com/seaweedfs/seaweedfs/blob/master/weed/util/log_buffer/log_read.go#L128
